workshop/modules/terraform-aws-eks-data-addons/emr-spark-operator.tf

```diff
@@ -0,0 +1,30 @@
+locals {
+
+  emr_spark_operator_yaml = yamlencode(merge(
+    yamldecode(templatefile("${path.module}/values/emr-spark-operator-values.yaml", { aws_region = local.region })),
+    try(yamldecode(var.emr_spark_operator_config.values[0]), {})
+  ))
+}
+
+resource "helm_release" "emr_spark_operator" {
```

Comment:
You need to use all the values that comes with helm release. Use the other helm release example.